.travis.yml

```diff
@@ -23,6 +23,6 @@ env:
     - ROS_DISTRO=melodic ROS_REPO=ros-testing
 
 install:
-  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+  - git clone --quiet --depth 1 https://github.com/ros-industrial/industrial_ci.git .industrial_ci -b master
```

Comment:
Changed install directory to `.industrial_ci` just to match the industrial_ci documentation, certainly doesn't make a big deal but I figured if people come looking to established repos such as this one for examples, it removes a friction point if we match the documentation as closely as is reasonable.

Reply:
What attention to detail! Thanks!